### PR TITLE
2724 material UI typography

### PR DESCRIPTION
**Pull request by OlgaPoddubnaia**

### CODE REVIEW

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java [hunk 1]
```diff
@@ -1,24 +1,49 @@
...
-        typographyFrame.h1Typography.has().styleClass("h1");
+        List<String> typographyClasses = Arrays.asList("MuiTypography-h1", "MuiTypography-h2", "MuiTypography-h3",
+                "MuiTypography-h4", "MuiTypography-h5", "MuiTypography-h6", "MuiTypography-subtitle1",
+                "MuiTypography-subtitle2", "MuiTypography-body1", "MuiTypography-body2", "MuiTypography-button",
+                "MuiTypography-caption", "MuiTypography-overline");
 
-        typographyFrame.subtitle2Typography.has().styleClass("subtitle2");
-        typographyFrame.subtitle2Typography.has().text("Subtitle 2");
+        for (Text typographyText : typographyTexts) {
+            Assert.assertTrue(textFields.contains(typographyText.getText()));
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:42]: please use static import for testNG asserts

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
+
+        List<String> textFields = Arrays.asList("Head 1", "Head 2", "Head 3", "Head 4", "Head 5", "Head 6",
+                "Subtitle 1", "Subtitle 2", "Body 1", "Body 2", "BUTTON TEXT", "Caption text", "OVERLINE TEXT");
 
-        typographyFrame.h1Typography.has().text("Head 1");
-        typographyFrame.h1Typography.has().styleClass("h1");
+        List<String> typographyClasses = Arrays.asList("MuiTypography-h1", "MuiTypography-h2", "MuiTypography-h3",
+                "MuiTypography-h4", "MuiTypography-h5", "MuiTypography-h6", "MuiTypography-subtitle1",
+                "MuiTypography-subtitle2", "MuiTypography-body1", "MuiTypography-body2", "MuiTypography-button",
+                "MuiTypography-caption", "MuiTypography-overline");
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:40]: need to create empty list for actual text fields and fill it below

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java (hunk 1 above)
Comment (OlgaPoddubnaia) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:42]: replaced import by static import

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java (hunk 2 above)
Comment (OlgaPoddubnaia) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:43]: created List actualTextFields

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/displaydata/TypographyPage.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+
+import java.util.List;
+
+public class TypographyPage extends WebPage {
+
+    @UI("//*[contains(@class,'MuiTypography-gutterBottom')]")
+    public static List<Text> typographyTexts;
+
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/displaydata/TypographyPage.java:13]: please delete extra line

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java [hunk 4]
```diff
@@ -1,24 +1,53 @@
...
+import static org.testng.Assert.assertEquals;
+
+/**
+ * To see an example of Typography web element please visit https://material-ui.com/components/typography/
+ */
+
 
 public class TypographyTests extends TestsInit {
 
+
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:24]: please delete extra line

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java [hunk 5]
```diff
@@ -1,24 +1,53 @@
...
+    @BeforeMethod
+    public void before() {
+        openSection("Typography");
+    }
+
+
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:30]: please delete extra line

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java [hunk 6]
```diff
@@ -1,24 +1,53 @@
...
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.displaydata.TypographyPage.typographyTexts;
+import static org.hamcrest.Matchers.containsString;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * To see an example of Typography web element please visit https://material-ui.com/components/typography/
+ */
+
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:21]: please delete extra line

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/displaydata/TypographyPage.java (hunk 3 above)
Comment (OlgaPoddubnaia) [reply jdi-light-material-ui-tests/src/main/java/io/github/com/pages/displaydata/TypographyPage.java:13]: deleted extra line 13

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java (hunk 4 above)
Comment (OlgaPoddubnaia) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:24]: deleted extra line 24

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java (hunk 5 above)
Comment (OlgaPoddubnaia) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:30]: deleted extra line 30

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java (hunk 6 above)
Comment (OlgaPoddubnaia) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:21]: deleted extra line 20

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java [hunk 7]
```diff
@@ -1,24 +1,50 @@
...
 
-        typographyFrame.subtitle2Typography.has().styleClass("subtitle2");
-        typographyFrame.subtitle2Typography.has().text("Subtitle 2");
+        for (Text typographyText : typographyTexts) {
+            actualTextFields.add(typographyText.getText());
+        }
 
-        typographyFrame.buttonTypography.has().styleClass("button");
-        typographyFrame.buttonTypography.has().text("BUTTON TEXT");
+        for (int i = 0; i < typographyTexts.size(); i++) {
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/TypographyTests.java:45]: it's ok

**Pull request merged**